### PR TITLE
DDF-5834 update plugins in DDF

### PR DESCRIPTION
**Pull request by stustison**
#### What does this PR do?
Upgrades almost all of the plugins used in DDF

#### Who is reviewing it? 

@SmithJosh @bakejeyner 

#### Ask 2 committers to review/merge the PR and tag them here.

@blen-desta
@garrettfreibott
@pklinef
@shaundmorris
@stustison

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/test/itests/test-itests-common/pom.xml [hunk 1]
```diff
@@ -176,6 +176,7 @@
             <plugin>
                 <groupId>org.apache.felix</groupId>
                 <artifactId>maven-bundle-plugin</artifactId>
+                <version>3.5.1</version>
```
Comment (shaundmorris) [inline distribution/test/itests/test-itests-common/pom.xml]: ❔ Do you recall what the previous version was being used here?

Comment (stustison) [reply distribution/test/itests/test-itests-common/pom.xml]: It was 3.5.1 for everything, but it was adding in an osgi.contraint that just wasn't needed and I couldn't figure out how to remove it. Setting it back to the older plugin got the tests to pass again and this is just a test bundle anyways.

Comment (pklinef) [reply distribution/test/itests/test-itests-common/pom.xml]: Is it worth a comment?

**Comment by stustison** [comment]
build now

File: distribution/test/itests/test-itests-common/pom.xml (hunk 1 above)
Comment (stustison) [reply distribution/test/itests/test-itests-common/pom.xml]: Maybe? I mean, I don't care if someone tries to update it again in the future. If it doesn't work, you figure it out really quick. If it does work, then it should get updated. I'm a little afraid that if I leave a comment on there that no one will ever try to upgrade it ever again.

Comment (pklinef) [reply distribution/test/itests/test-itests-common/pom.xml]: I was more thinking of a comment saying what did not work so they would know how to test if it does work in the future.  If it is an obvious failure then maybe it is not necessary.  Though I bet someone tries to ask you in the future why you did this.  A comment might save them that time.

**Pull request merged**